docs/source/_static/custom.css

```diff
@@ -0,0 +1,9 @@
+/* 24px margin from readthedocs theme */
+div.nblast {
+    margin-bottom: 19px !important;  /* padding has already 5px */
```

Comment:
so 
```
div.nbinput.nblast {
    margin-bottom: 19px;  /* padding has already 5px */
}
```

See to work for me, and the second `!important` seem to have no effect to me.